src/components/BoardCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import glamorous from 'glamorous'
+import { Subscribe } from 'unstated'
+
+import { spacing, colors, radii, fontSizes } from '../theme'
+import ModalContainer from '../containers/ModalContainer'
+import UpdateBoardModal from './UpdateBoardModal'
+import CreateBoardModal from './CreateBoardModal'
+import DeleteBoardModal from './DeleteBoardModal'
+
+const BoardCardContainer = glamorous.div({
```

Comment:
Maybe, we could make this a `Link` instead of a div, like this: `glamorous(Link)({ ... })`. That might be a good way to handle the text-decoration problem.